### ISSUE TITLE
GUI execute --script error

### ISSUE THREAD

**Issue by alain-rs** [os-fpga/FOEDAG#298]
A new issue introduce with the project sync, please test more your changes!

./build/bin/foedag --script  tests/Testcases/raygentop/raygentop.tcl 

# Created design source: RAYGENTOP
Adding VERILOG_2001 tests/Testcases/raygentop/raygentop.v
ERROR: Failed to add file: 0x5588eb49bcf8

**Comment by alain-rs** [comment]
Note this error was silent until I used the the ErrorMessage method in the add_source_file call. 
Something new is masking most error messages after the project sync PR.

**Comment by KochynVolodymyr** [comment]
I'll check this issue after #260.  Synchronization isn't finished completely.

**Comment by alain-rs** [comment]
I don't see this error anymore
